main.js

```diff
@@ -1,41 +1,60 @@
+let main = document.querySelector('.main');
+let preload = document.querySelector('.container');
+
+setTimeout (() => {
+	main.style.visibility = 'visible';
+  preload.style.display = 'none';
+},3000);
 let url = window.location.toString();
 
-let nickName = (link) => {
+let getNickName = (link) => {
   let arr = link.split('=');
   let nick = arr[1];
   if(!nick) {
     nick = "x1nax";
   }
   return nick;
 }
+nickName = getNickName(url);
```

Comment:
Codacy found an issue: ['nickName' is not defined.](https://app.codacy.com/gh/WebHeroSchool/jstask35/pullRequest?prid=6109934)